repos/fdbt-site/src/design/modules/_helpers.scss

```diff
@@ -27,3 +27,7 @@
 .dft-text-align-centre {
     text-align: center;
 }
+
+.dft-four-fifths-width {
```

Comment:
I don't think you need this, you can use the two-thirds. I think the design is just an oversight from Esmond.

Reply:
I think it looks better as four fifths as it fits better with the export button. Also what two thirds would you use? I can only find the column one which doesn't seem to work as expected